### PR TITLE
feat: allow upper case in doc-deploy-changelog

### LINKED ISSUE #496: Doc-deploy-changelog action does not work well with commit-style action when use-upper-case is set to True
By default `doc-deploy-changelog` creates a PR with title `chore: update CHANGELOG for vX.Y.Z`.
However, if one uses the `commit-style` with `use-upper-case = true` then the PR title does not follow the expected PR type.
A simple fix would be to add `use-upper-case` to the `doc-deploy-changelog` action.

### PR DESCRIPTION
**Pull request by SMoraisAnsys**
Closes #496

### CODE REVIEW

**Review by RobPasMue — APPROVED** [review]

**Comment by SMoraisAnsys** [comment]
Tested it while enabling the use of upper-case and it seems to work well

image

Should I also add a type to the commit added when creating the fragment ? We would have by default, "chore: adding changelog entry..." instead of "Adding changelog entry..."

File: doc-deploy-changelog/action.yml [hunk 1]
```diff
@@ -83,6 +83,22 @@ inputs:
     default: true
     type: boolean
 
+  use-upper-case:
+    description: >
+      Use of uppercase letters in the "type" field of:
+
+      - the PR created into main to add CHANGELOG changes;
```
Comment (Revathyvenugopal162) [inline doc-deploy-changelog/action.yml:90]:
```suggestion
      - the PR created into main to add CHANGELOG changes.
```

**Review by Revathyvenugopal162 — APPROVED** [review]

**Review by klmcadams — APPROVED** [review]

**Review by MaxJPRey — APPROVED** [review]
Thanks @SMoraisAnsys . I will be able to proceed with my upper case habits regarding PRs' titles.

File: doc-deploy-changelog/action.yml [hunk 2]
```diff
@@ -83,6 +83,22 @@ inputs:
...
+
+      - the commit created in the release branch to add the new
```
Comment (SMoraisAnsys) [inline doc-deploy-changelog/action.yml:92]:
```suggestion
      - The commit created in the release branch to add the new
```

**Comment by RobPasMue** [comment]
Can we merge this?

**Comment by SMoraisAnsys** [comment]
> Can we merge this?

Yes !

**Pull request merged**